src/demo-app/overlay/overlay-demo.ts

```diff
@@ -1,36 +1,32 @@
 import {
     Component,
-    ElementRef,
     ViewChildren,
     QueryList,
     ViewEncapsulation,
-    Directive,
     ViewChild,
     ViewContainerRef
 } from '@angular/core';
 import {Overlay, OverlayState, OVERLAY_PROVIDERS} from '@angular2-material/core/overlay/overlay';
 import {ComponentPortal, Portal} from '@angular2-material/core/portal/portal';
 import {TemplatePortalDirective} from '@angular2-material/core/portal/portal-directives';
-
-
-
-@Directive({selector: '[overlay-origin]'})
-class OverlayOrigin {
-  constructor(public elementRef: ElementRef) { }
-}
+import {
+    ConnectedOverlayDirective,
+    OverlayOrigin
+} from '@angular2-material/core/overlay/overlay-directives';
 
 
 @Component({
   moduleId: module.id,
   selector: 'overlay-demo',
   templateUrl: 'overlay-demo.html',
   styleUrls: ['overlay-demo.css'],
-  directives: [TemplatePortalDirective, OverlayOrigin],
+  directives: [TemplatePortalDirective, OverlayOrigin, ConnectedOverlayDirective],
```

Comment:
What do you think about putting `OverlayOrigin` and `ConnectedOverlayDirective` in one of our shouty arrays?  `MD_OVERLAY_DIRECTIVES`?